rasa/core/events/__init__.py

```diff
@@ -590,12 +591,12 @@ class ReminderScheduled(Event):
 
     def __init__(
         self,
-        action_name,
-        trigger_date_time,
-        name=None,
-        kill_on_user_message=True,
-        timestamp=None,
-        metadata=None,
+        action_name: Text,
+        trigger_date_time: datetime,
```

Comment:
what a weird type :-D

Reply:
Yep! Unfortunately it's declared as `class datetime`